### PR TITLE
[add-data][m] Updating the dataset

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made

- Updated the data from 2007 up until today

cc @Mikanebu

### CODE REVIEW

File: data/monthly.csv [hunk 1]
```diff
@@ -1,99 +1,734 @@
 Date,Total Equity,Domestic Equity,World Equity,Hybrid,Total Bond,Taxable Bond,Municipal Bond,Total
+2007-01-31,27441,5976,21464,5498,15047,12227,2820,47986
```
Comment (Mikanebu) [inline data/monthly.csv:2]: Why we have 4 rows per month with identical date, but different values, see previous version which has 1 row per month, could you check your script?

**Review by Mikanebu — CHANGES_REQUESTED** [review]
Check monthly.csv, it has duplication or extra rows, should be 1 row per month

File: data/monthly.csv (hunk 1 above)
Comment (gradedSystem) [reply data/monthly.csv:2]: @Mikanebu I have updated the data

**Comment by gradedSystem** [comment]
@Mikanebu should be fixed now

**Review by Mikanebu — APPROVED** [review]
LGTM!

**Pull request merged**